networkparams_test.go

```diff
@@ -70,13 +72,79 @@ func checkGenesisBlockRespectsNetworkPowLimit(
 	}
 }
 
+func checkPrefix(t *testing.T, prefix string, encoded, networkName string) {
+	if strings.Index(encoded, prefix) != 0 {
+		t.Logf(
+			"Address prefix mismatch for <%s>: expected <%s> received <%s>",
+			networkName,
+			prefix,
+			encoded,
+		)
+		t.FailNow()
+	}
+}
+
+func preFillArray(size int, value byte) []byte {
+	b := make([]byte, size)
+	for i := 0; i < size; i++ {
+		b[i] = value
+	}
+	return b
+}
+
+// checkInterval creates two corner cases defining interval
+// of all key values: [ xxxx000000000...0cccc , xxxx111111111...1cccc ],
+// where xxxx - is the encoding magic, and cccc is a checksum.
+// The interval is mapped to corresponding interval in base 58.
+// Then prefixes are checked for mismatch.
+func checkInterval(t *testing.T,
+	desiredPrefix string, keySize int, networkName string, magic [2]byte) {
+	// min and max possible keys
+	minKey := preFillArray(keySize, 0x00) // all zeroes
```

Comment:
Please don't use postfix comments.  I know there are a couple of instances of it, but almost nothing in the codebase does and consistency is more important.